opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js

```diff
@@ -207,17 +251,15 @@ class RiskTrackingPopover extends Component {
       R.pathOr([], ['logged_by']),
       R.mergeAll,
     )(node);
-    console.log('riskTrackingLoggedBy', node);
-
     const initialValues = R.pipe(
       R.assoc('entry_type', node?.entry_type || []),
       R.assoc('title', node?.name || ''),
       R.assoc('description', node?.description || ''),
       R.assoc('event_start', dateFormat(node?.event_start)),
       R.assoc('event_end', dateFormat(node?.event_end)),
-      R.assoc('logged_by', riskTrackingLoggedBy?.name || ''),
+      R.assoc('logged_by', riskTrackingLoggedBy?.name || []),
       R.assoc('status_change', node?.status_change || ''),
-      R.assoc('related_response', riskStatusResponse?.name || []),
+      R.assoc('related_responses', riskStatusResponse.map((value) => value.id) || []),
```

Comment:
@sachin755 correct me if I am wrong but if `riskStatusReponse` is empty, wont the result be an empty array already? Wouldn't it be appropriate to check if `riskStatusResponse` is not null instead? Just checking.